packages/sdk/src/CLRunner.ts

```diff
@@ -1947,7 +2035,7 @@ export class CLRunner {
                     },
                     labelAction: undefined,
                     logicResult: undefined,
-                    scoredAction: undefined
+                    scoredAction: undefined,
```

Comment:
Why the comma?

Reply:
Because the language supports it now and it's better.

Reply:
What's the advantage?  Easier add subsequent items when editing?

Reply:
Yea, developer productivity. I think mostly code reviews as you get to see the addition of something instead of the misleading edit of line above, but also makes it easier to manipulate text as copying line or moving within list always works.

Reply:
+1, love it.  This has been supported in C++ forever.